notas.py

```diff
@@ -45,6 +45,23 @@ def verificar(padron_web, email_web):
     return False
 
 
+def eliminar_columnas_comentadas(headers, alumno):
+    comentadas = []
+    for i in range(len(headers)):
+        if headers[i][0] == "_":
+            comentadas.append(i)
+    cant_ya_eliminadas = 0
+
+    resul_headers = headers[:]
+    resul_alumno = alumno[:]
+    for col_comentada in comentadas:
+        resul_headers.pop(col_comentada - cant_ya_eliminadas)
+        resul_alumno.pop(col_comentada - cant_ya_eliminadas)
+        cant_ya_eliminadas += 1
+    return resul_headers, resul_alumno
```

Comment:
Hago que devuelvan una copia en vez de modificar las mismas, por cualquier cosa que en el futuro querramos agregar algo que necesite de los valores originales, o lo que fuere

Reply:
En vez de hacer la lista de comentadas y luego iterarla no era más fácil hacer el pop en el primer for?

Reply:
Si, "pero no". Lo hice pura y exclusivamente para dejarlo ordenado nomás, y no eliminar sobre la lista que estoy iterando (me parece medio chancho jaja)

Reply:
Mi contribución (untested):

```python
def eliminar_columnas_comentadas(headers, alumno):
    hide = lambda l: [x for i, x in enumerate(l)
                      if not headers[i].startswith("_")]
    return hide(headers), hide(alumno)
```